### ISSUE TITLE
MO2 v2.0.7 nexusmods downloads not working correctly

### ISSUE THREAD

**Issue by Laenib** [TanninOne/modorganizer#1021]
Before I state my problem, here is what I did: I installed MO2 v2.0.7 along with a compatibility plugin for SSE from 'TESVSE Plugin for Mod Organizer 2 by Archost'. The only deviation from that is my moving the base directory of MO to the same drive as the executable and the game since my other drive can not handle the amount of mods I am planning to expand Skyrim on.

Now to the problem: Whenever I use a 'Download with Manager" Button on Nexusmods the download starts in MO2 but stops downloading at exactly 60915 Bytes (59KB) and aborts the download after a few tries. This size is independent of the mod that I am trying to download. I have pasted the log to: http://pastebin.com/fjtFt3z5
The ending of the log lines always says: "server replied: Requested Range Not Satisfiable (299)"
A number of fresh installs and deletion of all old and alternative Mod Managers have not fixed this problem for me so far.

I would greatly appreciate any insight into this issue.

**Comment by Deucezor** [comment]
Yes, and as you can see when you want to open a Nexus page from a mod or simply clic on the Earth Icon at the top which is supposed to open the Nexus page of Skyrim Special Edition it open wrong link : http://www.nexusmods.com/skyrimse instead of http://www.nexusmods.com/skyrimspecialedition. Maybe there is a connexion between these 2 bugs ?

And Loot doesnt work when you press the 'Sort' button over Plugins list, really hope someone will make this sweet Mod Manager work well.

PS: Sorry for my broken english

**Comment by BlayeeR** [comment]
In nexusinterface.cpp function from game_skyrimse.dll plugin gameShortName() is used to generate download link from nxm protocol. gameShortName() for Skyrim SE returns 'SkyrimSE' so it's used in urls. We can't change it to 'SkyrimSpecialEditon' because MO won't recognise our Skyrim SE game. I fixed it by checking return value of this function when it's used in nexusinterface.cpp. If it equals 'SkyrimSE' it creates urls directly for sse. 
[https://github.com/BlayeeR/modorganizer/tree/new_vfs_library](url)

**Comment by ThosRTanner** [comment]
No please dont' do that. If gameShortName doesn't work, there should be a gameNexusName added (or something similar). I know that's a lot more changes but it avoids nasty specials in the game manager code

**Comment by BlayeeR** [comment]
@ThosRTanner There wouldn't be a problem if name of the game on nexus was just skyrimse, I'll edit my fork tomorrow and will try to do it according to your sugestion

**Comment by LePresidente** [comment]
This should be fixed before release : https://github.com/TanninOne/modorganizer/issues/1026

**Comment by LePresidente** [comment]
@ThosRTanner if you going to use umbrella i suggest merging from TanninOne/modorganizer-umbrella as it is the most up to date.

**Comment by LePresidente** [comment]
will push a commit in a few using gameNexusName, just busy updating all the plugins to use it.

**Comment by LePresidente** [comment]
This has been fixed in my forks.

https://github.com/LePresidente/modorganizer-uibase/commit/d518a1c4a8ebc98a6cdf9a811bf89a236b321cff

https://github.com/LePresidente/modorganizer/commit/61edeb12f10707a03aea33e67ea6fd47990ee543

https://github.com/LePresidente/modorganizer-game_skyrimSE/commit/c70c5adbbcbe73b73557137bd1a4b9a7b1509b5b

should I push @ThosRTanner, is this kinda what you were getting at?

**Comment by LePresidente** [comment]
This has been fixed in 2.0.8b
